tests/Base.php

```diff
@@ -58,6 +58,24 @@ abstract class Base extends TestCase
         'WS:/v1/realtime:passed',
     ];
 
+    const QUERY_HELPER_RESPONSES = [
+        'title.equal("Spiderman","Dr. Strange")',
+        'title.notEqual("Spiderman")',
+        'releasedYear.lesser(1990)',
+        'releasedYear.greater(1990,1999)',
+        'name.search("john")',
+    ];
+
+    const PERMISSIONS_HELPER_RESPONSES = [
+        'read("any")',
+        'write("user:userid")',
+        'create("users")',
+        'update("guests")',
+        'delete("team:teamId/owner")',
+        'delete("team:teamId")',
+        'unique()',
```

Comment:
We should split this into `ID_HELPER_RESPONSES`

Reply:
done 👍